python/ray/tests/test_multi_node_2.py

```diff
@@ -222,6 +222,7 @@ def work(self, signal):
 
     ray.get(signal.send.remote())
     ray.get(work_handle)
+    del monitor
```

Comment:
Why is this necessary? Seems odd.

Reply:
It's referencing gcs, and we'd like it to stop monitor before stop gcs.